index.html

```diff
@@ -0,0 +1,156 @@
+            <span><a href="#">Mypage</a></span>
+            <span><a href="#">Logout</a></span>
+          </div>
+        </div>
+      </div>
+      <div class="mobile-menu-bg"></div>
+      <div class="hamburger-menu" id="hamburgerMenu">
+        <div class="bar1"></div>
+        <div class="bar2"></div>
+        <div class="bar3"></div>
+      </div>
+      <nav id="nav">
+        <div class="logo"><a href="index.html">
+          <img src="./assets/images/titleicon.png" class="linklogo-image"></a></div>
+        <ul class="menu">
+          <li><a href="about.html">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Blog</a></li>
+          <li id="codingCampaign"><a href="#home">Erriecha</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="title-and-description">
+        <h1 class="main-title"><q>Irreechaa  Love,Peace and Unity</q></h1>
+        <div class="hero-title">CELEBRATING IRREECHAA FESTIVAL 2023</div>
+        <div class="description-text">
+          A joyful celebration believing in value of openness and sharing. Creating a positive change with people from every corner of the countries is taking place in October in Ethiopian capital Finfinnee.
+        </div>
+        <h2 class="program-time">2023,10,15(saturday)-16(sunday)</h2>
+        <p class="program-place">
+          @Finfinnee, Hora Finfinnee and Bishooftuu, Hora Arsadii          
+        </p>
+      </section>
+      <section class="main-program">
+        <h3 class="main-program-title">Main Program</h3>
+        <div class="program-container">
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/assembly.png" alt="Lecture" />
+            </span>
+            <h4 class="title">Lecture</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/exhibition.png" alt="exhibition" />
+            </span>
+            <h4 class="title">Exibition</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/forum.png" alt="forum" />
+            </span>
+            <h4 class="title">Forum</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/workshop.png" alt="workshop" />
+            </span>
```

Comment:
- [ ] As suggested by the previous reviewer, all the cards' widths and heights must remain same at all screen sizes as in the Original design in Behance. and because we must follow the original design in all aspects as it is a project requirement. kindly consider fixing it.
your design
![qw](https://user-images.githubusercontent.com/96683843/226131220-624029f1-3e79-46ab-8e97-3098db280aa4.jpg)
![po](https://user-images.githubusercontent.com/96683843/226131261-e694e321-e75a-42fb-820f-b0c22b399880.jpg)


Behance design
![re](https://user-images.githubusercontent.com/96683843/226131229-03fb91a2-03d7-4b5e-b759-ab7a2c7c99b6.jpg)